GameData/RealismOverhaul/Engine_Configs/NAA75_110_Config.cfg

```diff
@@ -7,29 +7,29 @@
 //	A-6
 //	Redstone
 //
-//	Dry Mass: 670 Kg
+//	Dry Mass: 740 Kg
 //	Thrust (SL): ??? kN
-//	Thrust (Vac): 395.9 kN
+//	Thrust (Vac): 395.5 kN
 //	ISP: 218 SL / 249 Vac
-//	Burn Time: 145
```

Comment:
I shifted that change to the A-7 and dropped the A-6 time; Sutton actually has the A-7 listed as 121 nominal burn time, but because he does not list direct data for the A-6 I moved it there and used a closer to current value for the A-7.

Reply:
But the issue is that Mercury-Redstone was ethalox, so the extended burn time needs to be available on the A-6 (and the A-7 needs the full burntime of Jupiter-C / Juno I).

Reply:
(Further, you've changed the Hydyne variant to have the same vacuum Isp as the ethalox version, which seems very wrong. I'm guessnig Sutton is giving the figures for the A-7 running on ethalox, not Hydyne?)

Reply:
Hmm... I see what you mean... But, to do that, the A-6 would need to be dropped to some guessed thrust/isp number, or as it was already, the A-7 would need to be raised to some number. Otherwise, the A-7 would just be a direct downgrade to the A-6 minus the fuel type...

Reply:
Umm, Sutton has one less than a page remark about a Redstone engine using Hydyne, but the most detail it goes into is a "10% increase in thrust and Isp almost as much." I can put that into the A-7 config and switch the burn times?

Reply:
Ended up going with this as of right now, thoughts? (with the thrust rounded here, I just didnt want to add decimals. they're there in the config.)

A-6 - 121s burn, 218/249 Isp 395kN
A-7A - 143s burn, 218/249 Isp 395kN
A-7B - 155s burn, 239/265 Isp 409kN